specs/AdaptiveCardsXamarinSpec.md

```diff
@@ -0,0 +1,188 @@
+| Custom Element | ```(void) setCustomElementParser: (NSObject<ACOIBaseCardElementParser>) key: (NSString*)``` | P1 |
+| Custom Element | ```(void) setCustomElementRenderer: (ACRBaseCardElementRenderer*) key:(NSString *)``` | P1 |
+| Custom Element | ```(NSObject<ACOIBaseCardElementParser>*) getCustomElementParser: (NSString*)``` | P1 |
+| Custom Element | ```(ACRBaseCardElementRenderer*) getRenderer: (NSNumber *)``` | P1 |
+| Custom Action | ```(void) setActionRenderer: (ACRBaseActionElementRenderer) cardElementType: (NSNumber*)``` | P1 |
+| Custom Action | ```(ACRBaseActionElementRenderer*) getActionRenderer: (NSNumber*)``` | P1 |
+| Custom Action Set | ```(void) setActionSetRenderer: (id<ACRIBaseActionSetRenderer>)``` | P2 |
+| Custom Action Set | ```(id<ACRIBaseActionSetRenderer>) getActionSetRenderer``` | P2 | 
+| Resource Resolvers | ```(void) setResourceResolver: (NSObject<ACOIResourceResolver>) scheme: (NSString*)``` | P2 |
+| Resource Resolvers | ```(NSObject<ACOIResourceResolver>) getResourceResolverForScheme: (NSString*)``` | P2 |
+| Feature Registration | ```(void) addFeature: (NSString*) featureVersion: (NSString*)``` | P0 | 
+| Feature Registration | ```(NSString*) removeFeature: (NSString*)``` | P0 |
+| Feature Registration | ```(NSString*) getFeatureVersion: (NSString*)``` | P0 |
+| Resource Information | ```(NSArray<ACORemoteResourceInformation>*) remoteResourceInformation``` | P2 | 
+
+## Breaking changes
+
+There is one big breaking change to be introduced in the iOS platform due to the Xamarin bindings having a direct relationship with the name of the library, which requires changing the library name from ACFramework to AdaptiveCards.
+
+Besides that, no breaking changes should be introduced as the native platforms' code is only consumed and projected into C#.
+
+## Test Plan
+
+For the alpha and pre-release versions the testing will be performed manually
+
+## Release plan
+
+### Artifacts
+
+The artifacts to be released with this library will be:
+1. Source code for Xamarin bindings in the github repository
+1. Xamarin.Android nuget package
+1. Xamarin.iOS nuget package
+
+### Continuous Integration and Release
+
+To be able to release the previously mentioned nuget packages build definitions must be created. The current system we use for building the other packages allow us to build Xamarin.Android and Xamarin.iOS packages
+
+#### Pre-requisites
+
+The Xamarin packages require native libraries to be built beforehand, as we can't upload binaries into the repository, then the Android and iOS build definitions must be updated to copy the generated binaries into the xamarin projects folder. 
+
+#### Build definitions
+
+At least four build pipelines will be needed for this project, 2 pipelines for Continuous integration and 2 pipelines for the release. The pipelines for the release will be similar to the UWP and WPF pipelines that generate nuget packages and build against a visual studio solution/project.
+
+## Remaining Tasks
```

Comment:
>maining Tasks [](start = 5, length = 13)

[Team Spec Review] Rename to "Identified Tasks" -& actually list out all tasks needed to release the Alpha with cost estimates and a third column on "Required for Alpha release" ; These then land as TODO's in the github proposal's Description section as a checklist that we tick off prior to release. These costs also feed into the cost we are bubbling up to the rest of the team/management as noted in the ADO rollup task.